test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_onchange.test

```diff
@@ -30,7 +30,6 @@ accumulatorStart
 
```

Comment:
We have decided to defer this to a short future (see https://github.com/telefonicaid/fiware-orion/issues/1476#issuecomment-159562257). However at least a subscription_sequence_onchange_nocache.test.DISABLED with the original content should be created in this PR, to ease the future work (otherwise, we would need to do some archeology work in develop branch).